lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -88,13 +90,13 @@ class ReportRenderer {
    */
   _populateScore(element, score, scoringMode, title, description) {
     // Fill in the blanks.
-    const valueEl = element.querySelector('.lh-score__value');
+    const valueEl = DOM.find(element, '.lh-score__value');
```

Comment:
need to use the instance because we can't rely on this global being present.
so i guess let's drop the static and use it as `this._dom.find(...`

also, suggestion on alternative method signature above...